test/debugger/test-debug-break-on-uncaught.js

```diff
@@ -91,7 +91,10 @@ function runScenario(scriptName, throwsOnLine, next) {
 
         client.reqContinue(function(error) {
           assert.ifError(error);
-          setTimeout(assertHasPaused.bind(null, client), 100);
+          setTimeout(
+            assertHasPaused.bind(null, client),
+            common.platformTimeout(3000)
+          );
```

Comment:
(That will also have the benefit of *not* requiring a minimum of 6 seconds for the test to run.)